mmaction/models/recognizers/base.py

```diff
@@ -77,10 +77,11 @@ def average_clip(self, cls_score, num_segs=1):
 
         Using different averaging types ('score' or 'prob' or None,
         which defined in test_cfg) to computed the final averaged
-        class score.
+        class score. Only called in test mode.
 
         Args:
             cls_score (torch.Tensor): Class score to be averaged.
+            num_segs (int): Number of clips for each input sample.
```

Comment:
There is a mixture of `num_segments` and `num_segs`

Reply:
Maybe you mean a mixture of `clips` and `segs`? In addition to the mixture in this line, there has been a mixture of `clip` and `segs` in method name `average_clip` and function argument `num_segs`. 

I do understand this `num_segs` means test clips (both temporal samples from `twice_sample`/`num_clips` if `clip_len!=1`/`num_sample_positions ` and spatial samples from `ThreeCrop/TenCrop/MultiGroupCrop`) for one input sample.

I plan to modify this line to 
```
num_segs (int): Number of segments for each input sample.  Number of segments is calculated by: 
    1)twice_sample... 
    2) num_clips... 
    3) ThreeCrop... 
    4) num_sample_positions...
```

@dreamerlin Any suggestions?

Reply:
I think he means the mixture of variable name between `num_segments` and `num_segs`. we can unify it.

Reply:
... sure

Reply:
> I think he means the mixture of variable name between `num_segments` and `num_segs`. we can unify it.

Actually in this project, there are a lot of `num_segs` and `num_segments`. 

As far as I'm concerned, `num_segments` is more like a model definition related param. So, in `TSMHead` and `average_clip`, `num_segs` is better, just like `TSNHead/TPNHead`. 

Is this ok for you? @dreamerlin

Reply:
Agree. I think, it is hard to cause ambiguity for these two words.